websites/Y/YouTube/presence.ts

```diff
@@ -215,47 +214,42 @@ presence.on("UpdateData", async () => {
         ".style-scope.ytd-channel-name > a"
       ).textContent;
     }
-    const metaVideoElement = document.querySelector<HTMLLinkElement>(
-      "div[itemtype='http://schema.org/VideoObject'] > link"
-    );
-    if (
-      metaVideoElement &&
-      metaVideoElement.href ==
-        `https://www.youtube.com/watch?v=${document
-          .querySelector("#page-manager > ytd-watch-flexy")
-          .getAttribute("video-id")}`
-    ) {
-      const unlistedVideoElement = document.querySelector<HTMLMetaElement>(
-        "div[itemtype='http://schema.org/VideoObject'] > meta[itemprop='unlisted']"
-      );
+    const unlistedPathElement = document.querySelector<SVGPathElement>(
+        "g#privacy_unlisted > path"
+      ),
+      unlistedBadgeElement = document.querySelector<SVGPathElement>(
+        "h1.title+ytd-badge-supported-renderer path"
+      ),
       unlistedVideo =
-        unlistedVideoElement && unlistedVideoElement.content === "True";
-    }
-    const presenceData: PresenceData = {
-      details: vidDetail
-        .replace("%title%", finalTitle)
-        .replace("%uploader%", finalUploader),
-      state: vidState
-        .replace("%title%", finalTitle)
-        .replace("%uploader%", finalUploader),
-      largeImageKey: "yt_lg",
-      smallImageKey: video.paused
-        ? "pause"
-        : video.loop
-        ? "repeat-one"
-        : isPlaylistLoop
-        ? "repeat"
-        : "play",
-      smallImageText: video.paused
-        ? (await strings).pause
-        : video.loop
-        ? "On loop"
-        : isPlaylistLoop
-        ? "Playlist on loop"
-        : (await strings).play,
-      startTimestamp: timestamps[0],
-      endTimestamp: timestamps[1]
-    };
+        unlistedPathElement !== null &&
+        unlistedBadgeElement !== null &&
+        unlistedPathElement.getAttribute("d") ===
+          unlistedBadgeElement.getAttribute("d"),
+      presenceData: PresenceData = {
+        details: vidDetail
+          .replace("%title%", finalTitle)
+          .replace("%uploader%", finalUploader),
+        state: vidState
+          .replace("%title%", finalTitle)
+          .replace("%uploader%", finalUploader),
+        largeImageKey: "yt_lg",
+        smallImageKey: video.paused
+          ? "pause"
+          : video.loop
+          ? "repeat-one"
+          : isPlaylistLoop
+          ? "repeat"
+          : "play",
+        smallImageText: video.paused
+          ? (await strings).pause
+          : video.loop
+          ? "On loop"
+          : isPlaylistLoop
+          ? "Playlist on loop"
+          : (await strings).play,
+        startTimestamp: timestamps[0],
```

Comment:
Yo, can you remove these startTimestamp where is set for videos etc ty.

Reply:
Okay, I'll do it.
Sorry for not refactoring other code's parts in this PR I was focused to figure the problem up.